br/pkg/lightning/backend/kv/sql2kv.go

```diff
@@ -33,7 +33,9 @@ import (
 	"github.com/pingcap/tidb/expression"
 	"github.com/pingcap/tidb/meta/autoid"
 	"github.com/pingcap/tidb/parser/model"
-	"github.com/pingcap/tidb/parser/mysql"
+	"github.com/pingcap/tidb/parser/mysql" //nolint:goimports
+	// Import tidb/planner/core to initialize expression.RewriteAstExpr
+	_ "github.com/pingcap/tidb/planner/core"
```

Comment:
What if separate init only import-s?

Reply:
Let me try it again.

Reply:
```
[2022-07-03T15:57:37.755Z] br/pkg/lightning/backend/kv/sql2kv.go:36: File is not `goimports`-ed (goimports)

[2022-07-03T15:57:37.755Z] 	"github.com/pingcap/tidb/parser/mysql"

```

Reply:
if remove ```//nolint:goimports```

```
[2022-07-03T15:57:37.755Z] br/pkg/lightning/backend/kv/sql2kv.go:36: File is not `goimports`-ed (goimports)

[2022-07-03T15:57:37.755Z] 	"github.com/pingcap/tidb/parser/mysql"

```

I don't know what is it.